apps/projects/test/Projects.test.js

```diff
@@ -872,6 +848,46 @@ contract('Projects App', accounts => {
             await app.removeRepo(repoId, { from: repoRemover })
           })
         })
+
+        it('work can be accepted multiple times', async () => {
+          const bountyId = (await app.getIssue(repoId, 1))[0].toString()
+          //console.log(bountyId)
+          await bountiesEvents.contribute(root, bountyId, 5, { value: 5 })
+          await bountiesEvents.fulfillBounty(root, bountyId, [root], 'test')
+          await bountiesEvents.fulfillBounty(bountyManager, bountyId, [bountyManager], 'test2', { from: bountyManager })
+          await app.reviewSubmission(
+            repoId,
+            1,
+            0,
+            true,
+            'QmbUSy8HCn8J4TMDRRdxCbK2uCCtkQyZtY6XYv3y7kLgDl',
+            [9],
+            { from: bountyManager }
+          )
+
+          await app.reviewSubmission(
+            repoId,
+            1,
+            1,
+            true,
+            'QmbUSy8HCn8J4TMDRRdxCbK2uCCtkQyZtY6XYv3y7kLgDl',
```

Comment:
I would move most of these arbitrary magic values to constante to have a more manageable/less error prone test suite, but that's only a personal preference